### ISSUE TITLE
Unable to resize WebGL offscreen canvas when using RaF

### ISSUE THREAD

**Issue by haroldiedema** [electron/electron#16211]
* Output of `node_modules/.bin/electron --version`: v4.0.0
* Operating System (Platform and Version): Windows 10 Pro (x64)
* Output of `node_modules/.bin/electron --version` on last known working Electron version (if applicable): Not applicable. RaF wasn't available in workers prior to 4.x

**Expected Behavior**
When I resize an offscreen WebGL canvas that renders images in a Worker, I expect the renderer to keep producing images when scheduling frames using `requestAnimationFrame`.

**Actual behavior**
When the application starts, the "render loop" produces images as expected. Once the window is resized, and thus the offscreen canvas size needs updating as well, the image just stops updating while the "render loop" keeps running. There are no errors in the console.

I am unsure if this is a bug related to electron or chromium. I have tried to reproduce this on Chrome 70 and everything seems to work as expected there.

**To Reproduce**
```
git clone https://github.com/haroldiedema/materia-electron-4x-bugreport.git
yarn install
yarn start --debug
```

**Additional Information**
Typescript code is compiled when the application is started.

Comment or uncomment the line `this.renderer.setSize(width, height, false)` in `packages/Engine/Renderer/RendererThread.ts` at line 44.

The problem has something to do with `requestAnimationFrame`. When the `render()` method triggered from the main process and the `commit()` function is used on the `OffscreenCanvas`, everything works as expected.

So the actual bug: *Somehow RaF stops updating the actual canvas in the DOM when its size is being changed.*

\#edit: It appears that resizing the window back to its original size as it was when the rendering started, the canvas is updated again as if it was rendering frames all this time.

**Comment by haroldiedema** [comment]
@sofianguy I've decided to give this a go on the latest released Electron 5.x beta. This appears to have been solved with version v5.0.0-beta.4.

Perhaps it was a chromium related issue?

**Comment by sofianguy** [comment]
@haroldiedema thanks for the update. I am going to go ahead and close this as it is fixed in 5.0.x
